src/diff.js

```diff
@@ -110,6 +105,47 @@ function findFencedNodes(a) {
 // getting in the way of the diff algorithm when comparing characters.
 const FORCED_MATCH_CHAR = '@'
 
+const NODES_NAMES = [
```

Comment:
I think this name could be a little bit more descriptive. Maybe something like `BOUNDARY_NODES`?

Reply:
Yeah agree, the name is not clear